test/evergreen.yml

```diff
@@ -1427,6 +1427,36 @@ variables:
 # the number of runs adjusted to provide aproximately six hours of testing.
 #########################################################################################
 
+  - &format-stress-test-disagg-leader
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "format test disagg"
+        vars:
+          format_args: disagg.mode=leader runs.tables=1 runs.timer=3
+          times: 5
```

Comment:
wouldnt running it 5 times make it harder to look at the BBUI and connect failures? My view is run more tasks and each tasks runs this only once for now.

Reply:
The format test disagg would exit early once it encounters the first failure. So IMO, i think this is enough. What are your thoughts?

Reply:
true, makes sense